src/containers/HeaderAuth.jsx

```diff
@@ -24,6 +24,10 @@ export default class HeaderAuth extends Component {
   }
 
   login() {
+    console.log('Logging in');
+    // For deploy-preview, use the following Wildcam on staging details in config.json
+    // const appId = '17bdbeb57f54a3bf6344cf7150047879cfa1c8d5f9fd77d64923e6c81fe6e949';
+    // const redirectUri = 'https://preview.zooniverse.org/wge/';
     return Panoptes.oauth.signIn('http://localhost:3000/?env=staging#/');
```

Comment:
Minor query: if we want to deploy a preview to Master by the end of the day, would this need to be addressed with the 'correct' URLs & IDs for the preview version?